docs/configuration/examples.md

```diff
@@ -111,7 +111,8 @@ create-archive:
 
 If you need to generate local archive(s) during release syncing, you can utilise e.g. `pre-sync` action
 to place the commands necessary for the creation. You also have to include the archive(s) in the list of files to
-be moved to the dist-git repo so that it is then handled by Packit from there.
+be moved to the dist-git repo so that it is then handled by Packit from there. **Note:** make sure the archive(s)
+are excluded by the downstream `.gitignore`.
```

Comment:
I'm not sure it makes sense to include it here as I would say it is the intention for the files to be included if they are specified in the `files_to_sync`, WDYT?

Reply:
Hmm, kinda, but this is the only example for these. Let me try some rewording for this to be more specific about lookaside uploads